.travis.yml

```diff
@@ -17,7 +17,6 @@ php:
     - 5.5
     - 5.6
     - 7.0
-    - hhvm
 
 matrix:
```

Comment:
I can't understand what is the goal of your matrix here because the `SYMFONY_VERSION` env var is never used.